src/kayenta/edit/configJson.less

```diff
@@ -1,3 +1,14 @@
+.ace-editor {
+  div,
+  span {
+    font-family: Menlo, Monaco, Consolas, 'Courier New', monospace;
+  }
+
+  > div.ace_gutter {
+    z-index: 0;
+  }
+}
```

Comment:
do we have something that looks like this in `core`? or is this something specific to kayenta?

Reply:
ooooh shoot I meant to do something about this, will do something about this, will be better than this